specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-07-02-preview/managedClusters.json

```diff
@@ -4092,30 +4100,33 @@
           "description": "IPv4 or IPv6 address of the machine"
         },
         "family": {
-          "type": "string",
-          "readOnly": true,
-          "description": "To determine if address belongs IPv4 or IPv6 family",
-          "enum": [
-            "IPv4",
-            "IPv6"
-          ],
-          "x-ms-enum": {
-            "name": "IPFamily",
-            "modelAsString": true,
-            "values": [
-              {
-                "value": "IPv4",
-                "description": "IPv4 family"
-              },
-              {
-                "value": "IPv6",
-                "description": "IPv6 family"
-              }
-            ]
-          }
+          "$ref": "#/definitions/ipFamily",
+          "description": "To determine if address belongs IPv4 or IPv6 family"
         }
       }
     },
+    "ipFamily": {
+      "type": "string",
+      "readOnly": true,
```

Comment:
have to remove the read-only restriction, as the ref in network profile need to modify this property.

Reply:
And please add a description for this model definition